packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -240,32 +245,32 @@ export function makeScreenshotEditorComponent({ h, imageBuffer, dialog }: Factor
     return (
       <div class="editor">
         <style dangerouslySetInnerHTML={styles} />
-        <div class="canvasContainer" ref={canvasContainerRef}>
-          <div class="cropButtonContainer" style={{ position: 'absolute' }} ref={cropContainerRef}>
+        <div class="container--canvas" ref={canvasContainerRef}>
+          <div class="container--crop-btn" style={{ position: 'absolute' }} ref={cropContainerRef}>
```

Comment:
Here, `editor` is the "block" and the "element" is `canvas-container` and `crop-btn-container`

```suggestion
        <div class="editor__canvas-container" ref={canvasContainerRef}>
          <div class="editor__crop-btn-container" style={{ position: 'absolute' }} ref={cropContainerRef}>
```

`--` is used for a "modifier" (e.g. to change behavior/state)